consensus/core/src/authority_service.rs

```diff
@@ -226,6 +226,76 @@ impl<C: CoreThreadDispatcher> NetworkService for AuthorityService<C> {
             }
         }
 
+        // ------------ After processing the block, process the excluded ancestors
+        // ------------
```

Comment:
Comment looked weird after it was broken into 2 lines. 

```suggestion
        // After processing the block, process the excluded ancestors
```